frontend/src/container/LogsExplorerList/index.tsx

```diff
@@ -169,7 +169,9 @@ function LogsExplorerList({
 				!isFetching &&
 				logs.length === 0 &&
 				!isError &&
-				isFilterApplied && <EmptyLogsSearch />}
+				isFilterApplied && (
+					<EmptyLogsSearch dataSource={DataSource.LOGS} panelType="LIST" />
```

Comment:
NIT: use constant `DataSource`

Reply:
Aren't we already using DataSource Constant?